### PR TITLE
Make APCs explode (when turned on) after a random delay during a power grid check.

### PR DESCRIPTION
**Pull request by Simyon264**
## About the PR

Was mentioned that admins already do this manually. This PR just enforces this mechanically.

## Technical details

Adds a system which makes device explode when in a overloaded state. This can be adapted to be used for other devices which are not APCs.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
https://cdn.discordapp.com/attachments/1208920487267143711/1233757174031974410/Desktop_2024.04.27_-_14.28.14.03.mp4?ex=662e4168&is=662cefe8&hm=a5e47fa93679acb2992f4a86afd41ee1c023a3919cb6f275798beb252d493f4f&

**Changelog**

:cl: Simyon
- tweak: APCs will now explode during a power grid check when they are turned on.

### CODE REVIEW

File: Resources/Locale/en-US/power/overload.ftl [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿electrical-overload-system-buzz = buzzes loudly
```
Comment (TsjipTsjip) [inline Resources/Locale/en-US/power/overload.ftl:1]: `buzzes loudly` -> `buzzes`

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
Not tested in-game just yet, but the chat message showing up in the chatbox, along with the frequency of the buzzing, makes it quite annoying. Could you tone down the buzzing and look at how vendors send their advertisements? Those also don't appear in chat windows.

File: Resources/Locale/en-US/power/overload.ftl (hunk 1 above)
Comment (Simyon264) [reply Resources/Locale/en-US/power/overload.ftl]: Done.

**Review by arimah — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/power/overload.ftl [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿electrical-overload-system-buzz = buzzes
```
[the remark this answers is not in the document]
Comment (TsjipTsjip) [reply Resources/Locale/en-US/power/overload.ftl:1]: I did already request this to be changed from `buzzes loudly`. :p

Can workshop it if it happens again. Currently we already have a sort of standing fact that "Electrical stuff buzzing = Bad" through manual interactions, so I believe this is sufficient for now.

[the remark this answers is not in the document]
Comment (arimah) [reply Resources/Locale/en-US/power/overload.ftl:1]: Oh, I'm sorry! I missed that! I suppose we'll just keep it as "buzzes" and see what happens.

**Review by arimah — APPROVED** [review]
Works as intended, code looks fine. I'm happy with it. :)

**Review by TsjipTsjip — APPROVED** [review]

**Pull request merged**